jobs/JRRFS_RUN_MAXTMINRH

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+##########################################################
+# This job...
+# 
+########################################################
+
+date
+export PS4=' $SECONDS + ' 
+set -x
+
+if [ ${cyc} != "07" ] ; then
+    echo -e "\nWARNING: GETMINT only runs at 07z. Exiting.\n"
+    exit 0
+fi
+
+
+#######################################################
+## The following variable could be defined in the
+## submission script (the ecf script), if
+## not they will take the default values which is set
+## for the NCO running enviroment
+########################################################
+export RUN_ENVIR=${RUN_ENVIR:-nco}
+#
+##################
+## Set RUN
+##################
+export RUN=rtma3d
```

Comment:
The variable `RUN` is defined in the configuration file `config.sh`. If you want to change it, please set it in `config.sh` (not in the individual j-job or ex-script).